frontend/src/utils/filterHelpers.js

```diff
@@ -322,3 +328,44 @@ export function parseFiltersToPatch(params) {
     return acc;
   }, []);
 }
+
+const cleanupParameter = (param) => {
```

Comment:
rename it to prepareParameterForBackend to be consistent with the other function.